batavia/core/builtins.js

```diff
@@ -575,10 +574,18 @@ batavia.builtins.id = function() {
     throw new batavia.builtins.NotImplementedError("Builtin Batavia function 'id' not implemented");
 };
 
-batavia.builtins.input = function() {
-    throw new batavia.builtins.NotImplementedError("Builtin Batavia function 'input' not implemented");
+batavia.builtins.input = function(args) {
+    if (args.length > 1) {
+      throw new batavia.builtins.TypeError("input expected at most 1 arguments, got " + args.length);
+    }
+    if (args.length === 1) {
```

Comment:
There should also be a kwargs check here to make sure no kwargs have been provided.